scripts/generate_kardinal_version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this file? because I see there is a nix version

Reply:
we actually don't im gonna remove those

Reply:
get-docker-tag.sh is called by [generate_kardinal_version.nix](https://github.com/kurtosis-tech/kardinal/pull/280/files#diff-8176d17c009706d6eff9a19ebe156049b37ec9a9ee1699fd0682b70040a90bc6) I wonder is there is a way to replace it with the nix version of this script?

Reply:
ye i'm actually gonna embed the code for that directly in the generate nix